glib/src/object.rs

```diff
@@ -2513,6 +2513,11 @@ impl<T: ObjectType> Class<T> {
             }
         }
     }
+
+    /// Gets the contained type struct
+    pub fn inner(&mut self) -> &mut T::GlibClassType {
```

Comment:
Instead of that, can we get `AsRef<T>` and `AsMut<T>`? If not, naming them `as_ref()` and `as_mut()` would seem nicer nonetheless

Reply:
Good idea. That would probably be better.

Reply:
Though that does have the downside that it would probably require callers to specify the type, since it can't be inferred that `as_ref()` can only work with one type.

Reply:
But I think that's fine, since it may be more readable if code using this (which would probably mostly be implementations of `IsSubclassable`) is explicit about the type anyway.

I've changed this.